taxonomies/additives.txt

```diff
@@ -3587,6 +3587,7 @@ nl:E200, Sorbinezuur, Sorbinezuur
 nn:E200, sorbinsyre
 pl:E200, Kwas sorbowy, Kwas sorbowy, Kwas sorbinowy
 pt:E200, Ácido sórbico, Ácido sórbico, Sorbato
+ru:E200, Acid sorbic, sorbic acid, сорбиновая кислота, консервант сорбиновая кислота, Е200, Е-200
```

Comment:
we don't need the E200 at the end, as it's already at the beginning (there are other existing entries that have the E-number repeated, but that's a mistake).

we don't need E-200 either, as for additives, variations like E-200 are automatically generated